src/Raven.Server/Config/Categories/DatabaseConfiguration.cs

```diff
@@ -6,11 +6,23 @@ namespace Raven.Server.Config.Categories
 {
     public class DatabaseConfiguration : ConfigurationCategory
     {
-        [Description("The time to wait before canceling a database operation such as load (many) or query")]
-        [DefaultValue(5)]
-        [TimeUnit(TimeUnit.Minutes)]
-        [ConfigurationEntry("Databases.OperationTimeoutInMin", ConfigurationEntryScope.ServerWideOrPerDatabase)]
-        public TimeSetting OperationTimeout { get; set; }
+        [Description("The time in seconds to wait before canceling query operation")]
+        [DefaultValue(300)]
+        [TimeUnit(TimeUnit.Seconds)]
+        [ConfigurationEntry("Databases.QueryOperationTimeoutInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public TimeSetting QueryOperationTimeout { get; set; }
+
+        [Description("The time in seconds to wait before canceling terms indexing operation")]
+        [DefaultValue(300)]
+        [TimeUnit(TimeUnit.Seconds)]
+        [ConfigurationEntry("Databases.IndexTermsOperationTimeoutInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public TimeSetting IndexTermsOperationTimeout { get; set; }
+
+        [Description("The time in seconds to wait before canceling indexing terms operation")]
```

Comment:
Wrong description.

Reply:
The usage of this setting is only when deleting docs from the studio - not sure if it really deserves the configuration option.

Did you intent to introduce it in for delete / patch by query operations?

Reply:
Please see my question to you at http://issues.hibernatingrhinos.com/issue/RavenDB-9701
We need to modify this, I know.

Reply:
Answered there

Reply:
fixed in https://github.com/ravendb/docs/pull/407